src/itest/kotlin/nl/lifely/zac/itest/HistorieTest.kt

```diff
@@ -0,0 +1,112 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package nl.lifely.zac.itest
+
+import io.github.oshai.kotlinlogging.KotlinLogging
+import io.kotest.assertions.json.shouldContainJsonKey
+import io.kotest.core.spec.Order
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+import nl.lifely.zac.itest.client.ItestHttpClient
+import nl.lifely.zac.itest.config.ItestConfiguration.TEST_SPEC_ORDER_AFTER_TASK_COMPLETED
+import nl.lifely.zac.itest.config.ItestConfiguration.ZAC_API_URI
+import nl.lifely.zac.itest.config.ItestConfiguration.zaak1UUID
+import nl.lifely.zac.itest.util.shouldEqualJsonIgnoringExtraneousFields
+
+@Order(TEST_SPEC_ORDER_AFTER_TASK_COMPLETED)
+class HistorieTest : BehaviorSpec({
```

Comment:
Initial idea was to have it as a test for all histories, but I realized this would be too fragile and we have to change it with every test case we add.

Renamed to `ZaakHistorieTest`